### PR TITLE
feat(clustering) switch turn off wRPC also from CP

### PR DESCRIPTION
**Pull request by Suika-Kong**
and emit a warning

Fix FT-3077

### CODE REVIEW

**Comment by Suika-Kong** [comment]
remember to update this: https://github.com/Kong/kong/pull/9061

File: kong/init.lua [hunk 1]
```diff
@@ -1570,6 +1570,9 @@ end
 
 
 function Kong.serve_wrpc_listener(options)
+  if kong.configuration.legacy_hybrid_protocol then
+    return ngx.exit(404)
```
Comment (fffonion) [inline kong/init.lua:1574]: instead of return 404 here, we can also skip rendering the wrpc server block in nginx_template

**Comment by fffonion** [comment]
let's also use this option in wrpc test so that 1) we know it works 2) we know when it got removed

**Comment by Suika-Kong** [comment]
> let's also use this option in wrpc test so that 1) we know it works 2) we know when it got removed

https://github.com/Kong/kong/pull/9061

**Review by fffonion — APPROVED** [review]

File: kong/templates/nginx_kong.lua [hunk 2]
```diff
@@ -439,11 +439,14 @@ server {
         }
     }
 
+> if legacy_hybrid_protocol == "off" then
```
Comment (chronolaw) [inline kong/templates/nginx_kong.lua:442]: Here we need not to compare to `off`, it is a boolean, just use `if legacy_hybrid_protocol`.

You can refer to `legacy_worker_events`

Comment (Suika-Kong) [reply kong/templates/nginx_kong.lua:442]: This actually causes a bug. Here's the fix: #9085

**Pull request merged**